hudi-common/src/main/java/org/apache/hudi/common/table/log/InstantRange.java

```diff
@@ -174,6 +195,11 @@ public Builder nullableBoundary(boolean nullable) {
       return this;
     }
 
+    public Builder explicitInstants(Set<String> instants) {
+      this.explicitInstants = CollectionUtils.createImmutableSet(instants);
+      return this;
+    }
+
```

Comment:
This is only used by `ExplicitMatchRange`.  Should this setter be in the builder of `ExplicitMatchRange` only?

Reply:
We have only one builder here, no builder for specific impl clazz.